lib/usb-hid-msc-lib.robot

```diff
@@ -0,0 +1,35 @@
+*** Keywords ***
+Mount Bootable ISO
+    [Documentation]    Mounts a bootable ISO as flash USB. Currently
+    ...    only the Qubes OS ISO seems to work for the platform.
+    Mount Image    ${PIKVM_IP}    Qubes-R4.2.0-rc1-x86_64.iso
```

Comment:
This assumes we already have such image on PiKVM. Are we sure we do on each of them? Wouldn't it be easier to upload some smaller image (maybe DTS?) and mount it? You will need such keyword anyway for DTS tests, most likely.

Reply:
Resolved in [recent commit](https://github.com/Dasharo/open-source-firmware-validation/commit/9a7f15c18a71110e706347fdaa24543ddfc08a90)